cmd/update.go

```diff
@@ -537,6 +537,14 @@ func doUpdate(u *url.URL, lrTime time.Time, sha256Sum []byte, releaseInfo string
 		Checksum: sha256Sum,
 	}
 
+	if err := opts.CheckPermissions(); err != nil {
```

Comment:
```suggestion
	if err := opts.CheckPermissions(); err != nil {
```

🍰

Reply:
I am not sure what is this suggestion and I am not able to apply it as well.

Reply:
@vadmeste its just another way to say "very nice"